### PR TITLE
Release 13.3.7

### PR DESCRIPTION
**Pull request by fheinecke**

### CODE REVIEW

**Review by r0mant — APPROVED** [review]

**Review by tigrato — APPROVED** [review]

**Review by r0mant — CHANGES_REQUESTED** [review]
Actually, the changelog needs cleanup. It seems a lot of the changes are internal.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,24 @@
...
+* Added dynamic identity file reloading support for API Client [#31076](https://github.com/gravitational/teleport/pull/31076)
+* Fixed leaking connection monitor instances [#31042](https://github.com/gravitational/teleport/pull/31042)
+* Added calendar icon, export select style, and add type to validation rule (#30817) [#31036](https://github.com/gravitational/teleport/pull/31036)
+* Added OSS CTAs for support, access requests, and moderated sessions [#31030](https://github.com/gravitational/teleport/pull/31030)
+* Added support for using Amazon EC2 Instance Connect Endpoint to SSH into EC2 instances [#31021](https://github.com/gravitational/teleport/pull/31021)
+* Added support for IAM joining over reverse tunnel port [#31000](https://github.com/gravitational/teleport/pull/31000)
+* Added StateMessage and DashboardLink to List EICE [#30949](https://github.com/gravitational/teleport/pull/30949)
+* Added license flag to disable inaccessible features from being discoverable in the UI [#30936](https://github.com/gravitational/teleport/pull/30936)
+* Added access list members to the cache. [#30919](https://github.com/gravitational/teleport/pull/30919)
```
Comment (r0mant) [inline CHANGELOG.md:20]: These are internal changes.

**Review by r0mant — CHANGES_REQUESTED** [review]
And there's also no mention of the regression fix which is the primary reason we're publishing this release.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,24 @@
 # Changelog
 
+## 13.3.7 (08/29/23)
+
+* Added bblp custom theme and logos [#31149](https://github.com/gravitational/teleport/pull/31149)
```
Comment (zmb3) [inline CHANGELOG.md:5]: No need to mention this. Please remove.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,24 @@
...
+* Set `127.0.0.1:3080` as Vite default proxy target [#31148](https://github.com/gravitational/teleport/pull/31148)
+* `tsh request drop` behavior was restored to pre-v13.3.5 state where after dropping an access request, the role list will be based on the current role list of the user on the backend, not on the login state (as introduced in #30628) [#31136](https://github.com/gravitational/teleport/pull/31136)
```
Comment (zmb3) [inline CHANGELOG.md:7]: I would drop this too. It was not an intentional behavior change, and we're just restoring the behavior that only existed in one (buggy) release.

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,24 @@
...
+* Update headless modal to show both Reject and Cancel [#31135](https://github.com/gravitational/teleport/pull/31135)
+* Added support for proxy environment variables when dialing directly to the Kubernetes Cluster [#31133](https://github.com/gravitational/teleport/pull/31133)
+* Fixed the Oracle Database GUI Access flow on Windows Platform [#31129](https://github.com/gravitational/teleport/pull/31129)
+* Added OSS CTA for Auth Connectors (#30713) [#31083](https://github.com/gravitational/teleport/pull/31083)
+* Added dynamic identity file reloading support for API Client [#31076](https://github.com/gravitational/teleport/pull/31076)
+* Fixed leaking connection monitor instances [#31042](https://github.com/gravitational/teleport/pull/31042)
+* Added calendar icon, export select style, and add type to validation rule (#30817) [#31036](https://github.com/gravitational/teleport/pull/31036)
+* Added OSS CTAs for support, access requests, and moderated sessions [#31030](https://github.com/gravitational/teleport/pull/31030)
+* Added support for using Amazon EC2 Instance Connect Endpoint to SSH into EC2 instances [#31021](https://github.com/gravitational/teleport/pull/31021)
```
Comment (zmb3) [inline CHANGELOG.md:16]: Is this feature complete? I thought there was still a bit more work.

If so, we should probably avoid mentioning it in the changelog until everything required to use it is available.

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,24 @@
...
+* Added support for IAM joining over reverse tunnel port [#31000](https://github.com/gravitational/teleport/pull/31000)
+* Added StateMessage and DashboardLink to List EICE [#30949](https://github.com/gravitational/teleport/pull/30949)
```
Comment (zmb3) [inline CHANGELOG.md:18]: Remove this?

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,5 +1,24 @@
...
+* Added license flag to disable inaccessible features from being discoverable in the UI [#30936](https://github.com/gravitational/teleport/pull/30936)
```
Comment (zmb3) [inline CHANGELOG.md:19]: You can remove this too.

**Review by r0mant — APPROVED** [review]

**Review by zmb3 — APPROVED** [review]
Looks good after Roman's commit

**Pull request merged**